R/singleNode.getNodeRanksN.r

```diff
@@ -59,6 +59,10 @@
 singleNode.getNodeRanksN = function(n,G,p1,thresholdDiff,adj_mat,S=NULL,
                                     num.misses=NULL,verbose=FALSE,out_dir="",
                                     useLabels=FALSE,coords=NULL) {
+    
+    rownames(adj_mat) = names(G)
```

Comment:
do we need this?

Reply:
No, we don't - this was added so the Python-R code can work as NumPy matrices do not have row/column names.